crates/re_types/definitions/rerun/components/range1d.fbs

```diff
@@ -0,0 +1,19 @@
+include "arrow/attributes.fbs";
+include "python/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/attributes.fbs";
+
+namespace rerun.components;
+
+// ---
+
+/// A 1D range, specifying a lower and upper bound.
```

Comment:
Almost feels more like a datatype hum. But yeah definitely need it as a component here as is! Agreed with the notion that this is clearly not a blueprint type!
(not actionable)